tui/network.py

```diff
@@ -295,9 +493,12 @@ def specify_configuration(answers, txt, defaults):
             )
         ifaceName = conf_dict['config'].getInterfaceName(conf_dict['interface'])
         netutil.ifdown(ifaceName)
+        ifup_error = netutil.ifup(ifaceName) != 0
+        if conf_dict['config'].modev6 == NetInterface.Autoconf:
+            time.sleep(2)
 
         # check that we have *some* network:
-        if netutil.ifup(ifaceName) != 0 or not netutil.interfaceUp(ifaceName):
+        if ifup_error or not netutil.interfaceUp(ifaceName):
```

Comment:
this (and the extraction of a variable) seems unnecessary.

Reply:
The sleep is necessary to let some times for the autoconf of the interface.
It can be put inside the `netutil.ifup` method if you'd prefer.